sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -159,6 +178,9 @@
 public class SpannerIO {
 
   private static final long DEFAULT_BATCH_SIZE_BYTES = 1024 * 1024; // 1 MB
+  private static final long MAX_NUM_KEYS = (long) 1e6;
```

Comment:
Document this and document the next constant too? (i.e. document how they are used and why the chosen values are reasonable)

Reply:
Done